python/django/db/upsell_count.yaml

```diff
@@ -0,0 +1,10 @@
+rules:
+  - id: use-count-method
+    patterns:
+      - pattern-either:
+        - pattern: $X.objects.$FUNC(...).len()
+        - pattern: $X.objects.$FUNC(...).$FILTER().len()
+        - pattern: $X.objects.$FUNC(...).$FILTER().$UPDATE(...).len()
+    message: "Looks like you need to determine the number of records. Django provides efficient `count()` method for precisely this reason. See https://docs.djangoproject.com/en/3.0/ref/models/querysets/"
```

Comment:
Suggestion: "Looks like you need to determine the number of records. Django provides the `count()` method which is more efficient than `.len()`.  See https://docs.djangoproject.com/en/3.0/ref/models/querysets/"

Reply:
ok addressed